src/www/app/main.ts

```diff
@@ -59,28 +60,28 @@ function createServerRepo(
     console.debug('Detected development environment, using fake servers.');
     if (repo.getAll().length === 0) {
       repo.add(
-        shadowsocksConfigToAccessKey({
+        new OutlineServerAccessConfig({
```

Comment:
Revert to a factory function

Reply:
Yep, I'll use my class method from earlier